.github/workflows/samples-tools/cmd/affected/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+
+	"samples-tools/pkg/utils"
+)
+
+func main() {
```

Comment:
issue: combine affected and run-all into a single 'main' package, probably under just /cmd. Go best practices are to consolidate code files as much as possible and to avoid premature package-creation.

I would create a single CLI that has a `run-all` and `affected` subcommand. Then, if we build the binary during the session, we can get rid of ALL the extraneous code.

Reply:
All commands are merged into a single command.